guides/common/modules/con_network-boot-provisioning-workflow.adoc

```diff
@@ -60,11 +60,11 @@ If you use the discovery service, {Project} automatically detects the MAC addres
 Note that TCP port 8443 must be reachable by the {SmartProxy} to which the host is attached for {Project} to restart the host.
 
 PXE-less Provisioning::
-After you submit a new host request, you must boot the specific host with the boot disk that you download from {Project} and transfer using a USB port of the host.
+After you submit a new host request, you must boot the specific host with the boot disk that you download from {Project} and transfer by using a USB port of the host.
```

Comment:
I recommend rewriting "and transfer..." for clarity. Is the user _transferring_ a boot disk? Or mounting it by using the USB port? It sounds odd.

Perhaps break into two sentences.

Reply:
Would this work?
```suggestion
After you submit a new host request, you must boot the specific host with the boot disk that you download from {Project} and transfer by using an external storage device.
```

Reply:
Yes, that would work.